docker/oremda/Dockerfile

```diff
@@ -1,8 +1,16 @@
-FROM python:3.9
+FROM python:3.9-slim-bullseye
+
+RUN apt-get update && \
+  apt-get install -y \
+    git \
+    gcc && \
+  rm -rf /var/lib/apt/lists/*
 
 COPY . /oremda
 
-RUN pip install -e /oremda
+RUN pip install /oremda && \
```

Comment:
I'm not 100% sure, but I think `-e` might be needed in order for the bind mount of the oremda dir during development

Reply:
Fixed